src/sentry/tasks/integrations.py

```diff
@@ -51,3 +50,51 @@ def sync_assignee_outbound(external_issue_id, user_id, assign, **kwargs):
     else:
         user = User.objects.get(id=user_id)
     integration.get_installation().sync_assignee_outbound(external_issue, user, assign=assign)
+
+
+@instrumented_task(
+    name='sentry.tasks.integrations.sync_status_outbound',
+    queue='integrations',
+    default_retry_delay=60 * 5,
+    max_retries=5
+)
+@retry(exclude=(ExternalIssue.DoesNotExist, Integration.DoesNotExist))
+def sync_status_outbound(group_id, external_issue_id, **kwargs):
+    try:
+        group_status = Group.objects.filter(
+            id=group_id,
+            status__in=[GroupStatus.UNRESOLVED, GroupStatus.RESOLVED],
```

Comment:
we're not doing anything for ignore at least for now, so filtered down to just these two